### ISSUE TITLE
Inconsistent docker image names between Makefile and CI

### ISSUE THREAD

**Issue by bnevis-i** [edgexfoundry/device-coap-c#35]
In the Makefile, this project builds a docker-device-coap-c container.

```
device_coap:
	    docker build \
	        -f scripts/Dockerfile.alpine \
	        --label "git_sha=$(GIT_SHA)" \
	        -t edgexfoundry/docker-device-coap-c:${GIT_SHA} \
	        -t edgexfoundry/docker-device-coap-c:${VERSION}-dev \
            .
```

But the Jenkinsfile builds the "device-coap" container.
https://github.com/edgexfoundry/device-coap-c/blob/main/Jenkinsfile

We should update the Makefile to use the new standard container name.

**Comment by lenny-intel** [comment]
@bnevis-i , good catch. Apparently a miss when we dropped the `docker-` and `-c` from the names.

**Comment by iain-anderson** [comment]
closed #37
